docs/secure-connections/sauce-connect/advanced/sc-kgp.md

```diff
@@ -0,0 +1,46 @@
+---
+id: sc-kgp
+title: KGP
+sidebar_label: Tunneling Protocol
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+This document provides information about KGP - the tunneling protocol used by Sauce Connect Proxy&trade;
+to assist customer network and security engineering teams to better understand Sauce Connect Proxy&trade; networking.
+It's recommended to become familiar with [Sauce Connect Proxy&trade; architecture](/secure-connections/sauce-connect/advanced/sc-architecture)
+first.
+
```

Comment:
Before this section, I would add a "what you'll need"... so something like 

## What You'll Need
- Have the Sauce Connect Proxy client [downloaded](/secure-connections/sauce-connect/installation) to your local machine
- Review [Sauce Connect Proxy architecture](/secure-connections/sauce-connect/advanced/sc-architecture)